source/module_base/test/formatter_contextfmt_test.cpp

```diff
@@ -0,0 +1,369 @@
+#include "../formatter.h"
+#include <gtest/gtest.h>
+
```

Comment:
Please add some comments of the unit tests. Thanks!

Reply:
Not at all, I have completed adding short instructions in each unittest file